sdk/core/Azure.Core/src/Shared/IUtf8JsonSerializable.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.Text.Json;
+
+namespace Azure.Core
+{
+    internal interface IUtf8JsonSerializable
```

Comment:
Nope, these files do not belong here.

Reply:
Where should they go?

Reply:
We shouldn't copy any code generator shared code into azure-sdk-for-net. I suspect you don't need most of what's in these files anyway.

Reply:
Could we maybe setup a merge relationship between the AutoRest repo so things will be kept in sync?